packages/app-page-builder/src/blockEditor/config/editorBar/SaveBlockButton/SaveBlockButton.tsx

```diff
@@ -9,6 +9,9 @@ import { EditorBar } from "~/editor";
 import { useEventActionHandler } from "~/editor/hooks/useEventActionHandler";
 import { useBlock } from "~/blockEditor/hooks/useBlock";
 import { SaveBlockActionEvent } from "~/blockEditor/config/eventActions/saveBlock/event";
+import { useUI } from "~/editor/hooks/useUI";
+import { setDisplayModeMutation } from "~/editor/recoil/modules";
+import { DisplayMode } from "~/types";
```

Comment:
@neatbyte-vnobis let's please hide this low-level code inside of the [useDisplayMode](https://github.com/webiny/webiny-js/blob/990c3ab1b6577fb6f438c5ba0beacd6435e96f2d/packages/app-page-builder/src/editor/hooks/useDisplayMode.tsx#L12) hook. Just add a `setDisplayMode` function and hide the internals there in one place.